stubs/click-web/click_web/resources/cmd_exec.pyi

```diff
@@ -0,0 +1,79 @@
+import logging
+
+from flask import Response
+
+from .input_fields import FieldId
+
+logger: logging.Logger | None
+
+HTML_HEAD: str
+HTML_TAIL: str
+
+class Executor:
+    RAW_CMD_PATH: str
+
+    def __init__(self) -> None: ...
+    def exec(self, command_path: str) -> Response: ...
+    def _exec_raw(self, command: list[str]) -> Response: ...  # undocumented
+    def _exec_html(self, command_path: str) -> Response: ...  # undocumented
+    def _run_script_and_generate_stream(self) -> None: ...  # undocumented
+    def _create_cmd_header(self, commands: list[CmdPart]) -> str: ...  # undocumented
+    def _create_result_footer(self) -> str: ...  # undocumented
+
+def _get_download_link(field_info: FieldFileInfo) -> str: ...  # undocumented
+
+class CommandLineRaw:
+    def __init__(self, script_file_path: str, command: str) -> None: ...
+    def append(self, part: str, secret: bool = False) -> None: ...
+    def get_commandline(self, obfuscate: bool = False) -> list[str]: ...
+    def get_download_field_infos(self) -> list[FieldInfo]: ...
+    def after_script_executed(self) -> None: ...
+
+class CommandLineForm:
+    def __init__(self, script_file_path: str, commands: list[str]) -> None: ...
+    def append(self, part: str, secret: bool = False) -> None: ...
+    def get_commandline(self, obfuscate: bool = False) -> list[str]: ...
+    def get_download_field_infos(self) -> list[FieldInfo]: ...
+    def after_script_executed(self) -> None: ...
+
+def _get_python_interpreter() -> str: ...
+
+class CmdPart:
+    def __init__(self, part: str, secret: bool = False) -> None: ...
+
+class FormToCommandLineBuilder:
+    def __init__(self, command_line: CommandLineForm) -> None: ...
+    def add_command_args(self, command_index: int) -> None: ...
+    @staticmethod
+    def _is_option(cmd_option: str) -> bool: ...
```

Comment:
there are a lot of underscore-prefixed methods included in this PR - are they commonly used by the users of the library, or can we omit the ones that aren't part of the public API?

Reply:
https://github.com/python/typeshed/actions/runs/12211488466/job/34068824746?pr=13208#step:6:194
```
stub_uploader.metadata.InvalidRequires: Expected dependency Jinja2 to be present in the stub_uploader allowlist
```

Reply:
There are some strange convention in `typeshed`'s CICD in specifying what is public/private API. I'm too lazy to dig where I should register private names to make CICD happy. You are free to reject/patch this PR.